### PR TITLE
Add feedback api

### PR DESCRIPTION
**Pull request by richiebres**

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://15-bb-apidocs.surge.sh

File: feedback/get-list.adoc [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    "path":   [
+      {
+        "name": ":app_id:",
+        "type": "string",
+        "desc": 'App identifier',
+        "req":  true,
+        "cue":  'Enter an app id'
+      },
+      {
+        "name": ":Feedback_id:",
```
Comment (bbewan) [inline feedback/get-list.adoc]: I think this should be all lowercase: `feedback_id`.

File: feedback/get-list.adoc [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+== Definition
+
+[.definition.placeholder]
+Placeholder for the definition; auto-generated from JSON above.
+
+Use this method to retrieve a list of feedback received for an app.
+
+This method is paginated. By default 10 items are returned in the
+response. You can request up to 100 items at a time using the `pagesize`
```
Comment (bbewan) [inline feedback/get-list.adoc]: Is it `pagesize` or `pageSize` (as listed in the query params)? Capitalization should be consistent.

File: feedback/get-list.adoc [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+----
+
+== Definition
+
+[.definition.placeholder]
+Placeholder for the definition; auto-generated from JSON above.
+
+Use this method to retrieve a list of feedback received for an app.
+
+This method is paginated. By default 10 items are returned in the
```
Comment (bbewan) [inline feedback/get-list.adoc]: Add a comma after `default`: `By default, 10 items...`

File: feedback/get-list.adoc [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+.Response
+[source,json]
+[
+  {
+    "_id": "58a4e07838704cb2eacd7ce4",
+    "app_name": "2048 iOS App",
+    "platform": "ios"
+  },
+  {
```
Comment (bbewan) [inline feedback/get-list.adoc]: We should remove the empty block, and the comma from the previous line.

File: feedback/get-list.adoc (hunk 1 above)
Comment (bbewan) [reply feedback/get-list.adoc]: Also, why would an API client need to pass `feedback_id` here?

File: feedback/README.adoc [hunk 5]
```diff
@@ -0,0 +1,8 @@
+= Feedback
+
+This section covers the API methods for retrieving information about
+feedback received for your app through the buddybuild SDK:
```
Comment (bbewan) [inline feedback/README.adoc]: Turn `feedback` into a link to the docs covering the Feedback reporter:

```
link:https://docs.buddybuild.com/sdk/feedback_reporter.html[feedback]
```

Comment (richiebres) [reply feedback/README.adoc]: 👍

File: feedback/get-list.adoc (hunk 1 above)
Comment (richiebres) [reply feedback/get-list.adoc]: removed

File: feedback/get-list.adoc (hunk 3 above)
Comment (richiebres) [reply feedback/get-list.adoc]: 👍

File: feedback/get-list.adoc (hunk 2 above)
Comment (richiebres) [reply feedback/get-list.adoc]: 👍

File: feedback/get-list.adoc (hunk 4 above)
Comment (richiebres) [reply feedback/get-list.adoc]: 👍

**Comment by bbdocs** [comment]
See the website here: http://15-bb-apidocs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://15-bb-apidocs.surge.sh

**Pull request merged**